emmet-core/emmet/core/qchem/calc_types/utils.py

```diff
@@ -24,7 +24,7 @@
     "SOLVENT=WATER": "water",
     "SOLVENT=THF": "thf",
```

Comment:
Can we get rid of these? I'm handling the case of named solvents on the front-end. Let's only include the custom SMD params. e.g.

```python
synonyms = {
    "DIELECTRIC=18,500;N=1,415;ALPHA=0,000;BETA=0,735;GAMMA=20,200;PHI=0,000;PSI=0,143": "3:7 EC:EMC"
    "DIELECTRIC=18,500;N=1,415;ALPHA=0,000;BETA=0,735;GAMMA=20,200;PHI=0,000;PSI=0,000": "3:7 EC:EMC"
}
```

Reply:
Done!